modal/cli/app.py

```diff
@@ -111,3 +111,60 @@ async def stop(
         app_id = await get_app_id_from_name.aio(name, env, client)
     req = api_pb2.AppStopRequest(app_id=app_id, source=api_pb2.APP_STOP_SOURCE_CLI)
     await client.stub.AppStop(req)
+
+
+@app_cli.command("history", no_args_is_help=True)
+@synchronizer.create_blocking
+async def history(
+    app_id: str = Argument("", help="Look up an App's deployment history by its ID"),
+    *,
+    env: Optional[str] = ENV_OPTION,
+    json: bool = False,
+):
+    """Show App Deployment history
+
+    **Examples:**
+
+    Get the history based on an app ID:
+
+    ```bash
+    modal app history ap-123456
+    ```
+
+    """
+    if not app_id:
+        raise UsageError("Must pass either an ID or a name.")
```

Comment:
I think we should support name-based lookup -- but if we don't, should remove this.